src/components/Form/FormFields/Utils.ts

```diff
@@ -32,6 +32,7 @@ export type FormFieldBaseProps<T> = {
   error?: FieldError;
   onFocus?: (event: FocusEvent<HTMLInputElement>) => void;
   onBlur?: (event: FocusEvent<HTMLInputElement>) => void;
+  help?: boolean;
```

Comment:
```suggestion
```

Reply:
but you can pass it directly to the PhoneNumberFormField instead right?

Reply:
> but you can pass it directly to the PhoneNumberFormField instead right?

yes I am doing